models/setting.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (sequelize, DataTypes) =>
```

Comment:
add a follow up ticket for implementing a migration.  We can follow up about it later.